.github/workflows/pytest-gpu.yml

```diff
@@ -59,7 +62,9 @@ jobs:
 
     - name: Test with pytest
       run: |
-        pgaccelinfo
+        if [ "${{ matrix.name }}" == 'pytest-gpu-acc' ]; then
```

Comment:
why not always printing it? It doesn't hurt...
Well, maybe we need to print it for openmp as well?
Why? It **may** reveal any cuda installation problems.
I would keep it.

Reply:
I mean I would not have it printed conditionally.

Reply:
That command doesn't work on machines without acc. Hence the conditional.